docs/custom-flows/after-auth-flows.mdx

```diff
@@ -0,0 +1,260 @@
+
+`clerk-js` applies default route paths for tasks. Here's the mapping for each task type:
+
+- `orgs` from [Disable personal workspaces](/docs/organizations/overview#disable-personal-workspaces): `/sign-in/tasks/add-organization` or `/sign-up/tasks/add-organization`
+
+## Disable personal workspace flow
+
+The below snippets demonstrate how to build after-auth flows as custom flows, using [Disable personal workspaces](/docs/organizations/overview#disable-personal-workspaces) as an example.
+
+<Steps>
+  ### Build custom page to resolve task
+
+  Create a custom page that resolves the user's pending session task. For this example, we'll build a page where users can create and select an organization:
+
+  <CodeBlockTabs options={["index.html", "main.js"]}>
+    ```html {{ filename: 'index.html' }}
+    <!doctype html>
+    <html lang="en">
+      <head>
+        <meta charset="UTF-8" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <title>Clerk + JavaScript App</title>
+      </head>
+      <body>
+        <div id="app"></div>
+
+        <h1>Create an organization</h1>
+        <form id="create-organization">
+          <label for="name">Name</label>
+          <input id="name" name="name" />
+          <button>Create organization</button>
+        </form>
+
+        <script type="module" src="/src/main.js" async crossorigin="anonymous"></script>
+      </body>
+    </html>
+    ```
+
+    ```js {{ filename: 'main.js', mark: [[29, 32]] }}
+    import { Clerk } from '@clerk/clerk-js'
+
+    const pubKey = import.meta.env.VITE_CLERK_PUBLISHABLE_KEY
+
+    if (!pubKey) {
+      throw new Error('Add your VITE_CLERK_PUBLISHABLE_KEY to .env file')
+    }
+
+    const clerk = new Clerk(pubKey)
+    await clerk.load()
+
+    if (clerk.isSignedIn) {
+      const form = document.getElementById('create-organization')
+
+      form.addEventListener('submit', function (e) {
+        e.preventDefault()
+
+        const inputEl = document.getElementById('name')
+
+        if (!inputEl) {
+          // ... handle empty input
+          return
+        }
+
+        clerk
+          .createOrganization({ name: inputEl.value })
+          .then(async (organization) => {
+            // Select organization to the current session
+            await clerk.setActive({ organization: organization.id })
+
+            // Check for and navigate to any pending tasks after setting the active organization
+            await clerk.navigateToTask()
+          })
+          .catch((error) => console.log('An error occurred:', error))
+      })
+    } else {
+      // If there is no active user, mount Clerk's <SignIn />
+      document.getElementById('app').innerHTML = `
+        <div id="sign-in"></div>
+      `
+
+      const signInDiv = document.getElementById('sign-in')
+
+      clerk.mountSignIn(signInDiv)
+    }
+    ```
+  </CodeBlockTabs>
+
+  ### Handle `pending` sessions after sign-in
```

Comment:
For tomorrow - TODO:
- Update "custom page" snippet to include Next.js as example 
- Add sign-up snippet 
- Update `navigateToTask` reference